firebase-appdistribution/test-app/src/main/java/com/googletest/firebase/appdistribution/testapp/AppDistroTestApplication.kt

```diff
@@ -7,8 +7,8 @@ class AppDistroTestApplication : Application() {
         super.onCreate()
 
         // Perform any required trigger initialization here
-        ScreenshotDetectionFeedbackTrigger.initialize(this, R.string.terms_and_conditions);
-        NotificationFeedbackTrigger.initialize(this);
+        ScreenshotDetectionFeedbackTrigger.initialize(this, R.string.terms_and_conditions)
+//        NotificationFeedbackTrigger.initialize(this);
```

Comment:
Remove entirely? But maybe not - I think we should keep this trigger in the test app... (see below)

Reply:
I put it back based on your feedback below.